examples/generate_image_dataset/user_config.yaml.example

```diff
@@ -3,16 +3,17 @@
 #
 
 SPEAR:
-  LAUNCH_MODE: "standalone_executable"
+  INSTANCE:
+    LAUNCH_MODE: "standalone"
 
-  STANDALONE_EXECUTABLE: "/Users/mroberts/Downloads/SpearSim-Mac-Shipping/SpearSim-Mac-Shipping.app"
+    STANDALONE: "/Users/mroberts/Downloads/SpearSim-Mac-Shipping/SpearSim-Mac-Shipping.app"
 
-  PAKS_DIR: "/Users/mroberts/Downloads/spear-paks"
+    PAKS_DIR: "/Users/mroberts/Downloads/spear-paks"
 
-  CUSTOM_COMMAND_LINE_ARGUMENTS: []
-  RENDER_OFFSCREEN: False
-  WINDOW_RESOLUTION_X: 1024
-  WINDOW_RESOLUTION_Y: 1024
+    COMMAND_LINE_ARGS:
```

Comment:
Why do we have `COMMAND_LINE_ARGS` and `CUSTOM_COMMAND_LINE_ARGUMENTS`? I don't think `renderoffscreen` should be the default.